datajoint/fetch.py

```diff
@@ -20,46 +25,57 @@ def prepare_attributes(relation, item):
         raise DataJointError("Index must be a slice, a tuple, a list, a string.")
     return item, attributes
 
-class FetchQuery:
+def copy_first(f):
```

Comment:
`copy.copy` does not copy the dict `behavior`. The function `deepcopy` would copy the relation as well. That's why I introduced a copy constructor and used a decorator to tell the function, that I'd like `self` copied. Having the elements of `behavior` as object attributes wouldn't solve the problem, since some of the values of `behavior` are lists.